### DISCUSSION TITLE
Person, Birthday attribute

### DISCUSSION THREAD

**Discussion by theantnest** [photoprism/photoprism#3020] [Ideas]
A cool feature would be adding a birthdate field for tagged people, and on that day, show pictures of that person on the front page.

Similar to, "On this day" suggestion.

**Comment by graciousgrey** [comment]
This is something we plan to do, see: https://github.com/photoprism/photoprism/issues/1552

**Reply by theantnest** [reply]
Ha! Why am I not surprised that you guys are already on top of that?

Amazing, thanks and happy holidays.
